lib/internal/modules/esm/loader.js

```diff
@@ -3,7 +3,7 @@
 const {
   FunctionPrototypeBind,
   ObjectSetPrototypeOf,
-  SafeMap,
+  SafeMap
```

Comment:
seems like this could be reverted, since this file is not otherwise touched?

Reply:
Well spotted, added.